nanopc-t6/fit_args.sh

```diff
@@ -0,0 +1,163 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Rockchip Electronics Co., Ltd
+#
+# SPDX-License-Identifier: GPL-2.0
```

Comment:
The GPL license is somewhat problematic. Besides, you are copying the whole file ([link](https://raw.githubusercontent.com/friendlyarm/uboot-rockchip/d2512f90e9756130358771d4feb1d66d13dd6a90/arch/arm/mach-rockchip/fit_args.sh)) only to change `TEE_OFFSET`. Isn't there another way to specify the TEE load address?